containers/themes/ThemesSection.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import { c } from 'ttag';
+import {
+    useMailSettings,
+    useEventManager,
+    useApiWithoutResult,
+    SubTitle,
+    Alert,
+    ThemeCards,
+    ConfirmModal,
+    TextArea
+} from 'react-components';
+import { updateTheme } from 'proton-shared/lib/api/mailSettings';
+import { themeDark, themeLight, themeBlue, themeCustom } from './availableThemes.js';
+
+const availableThemes = [themeDark, themeLight, themeBlue, themeCustom];
+
+const getThemeName = (Theme) => {
+    if (Theme) {
+        if (/\/* light-theme \/*/.test(Theme)) {
```

Comment:
With `startsWith` instead of the RegExp please.

Reply:
Done!